lib/ansible/module_utils/facts/hardware/darwin.py

```diff
@@ -85,7 +85,8 @@ def get_cpu_facts(self):
 
     def get_memory_facts(self):
         memory_facts = {
-            'memtotal_mb': int(self.sysctl['hw.memsize']) // 1024 // 1024
+            'memtotal_mb': int(self.sysctl['hw.memsize']) // 1024 // 1024,
+            'memfree_mb': 0,
```

Comment:
I added this so that we always return a value for `ansible_memfree_mb`. In the current code in `devel`, if `vm_stat` isn't found, that will set `vm_stat_command` to `None`, which gets passed to `run_command()`. That results in the following error:

```
localhost | FAILED! => {
    "changed": false,
    "cmd": null,
    "msg": "Argument 'args' to run_command must be list or string",
    "rc": 257
}
```

So I kept the same behavior, which is setting `vm_stat_command` to `None`, but I added a conditional that avoids calling `run_command()` with a value of `None`.

I could probably do this differently, but that's the reasoning behind this change.